sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/KalixServer.scala

```diff
@@ -142,48 +146,47 @@ object KalixServer {
     }
   }
 
-  class ActionCreationContextFactoryBean(loco: ThreadLocal[ActionCreationContext])
-      extends FactoryBean[ActionCreationContext] {
+  abstract class ThreadLocalFactoryBean[T: ClassTag] extends FactoryBean[T] {
```

Comment:
All in one!

Introducing to avoid having to define each time a thread local and a factory bean.